### PR TITLE
docs: Fixed "Edit on GitHub" link for plugin

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Fixed sphinx theme to navigate "Edit on Github" link to locate correct
plugin source in GitHub repo.

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE 
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/plugin_doc_link_fix.yml

### CODE REVIEW

**Comment by Akasurde** [comment]
Fixes: https://github.com/ansible/ansible/issues/41425

File: docs/templates/cli_rst.j2 [hunk 1]
```diff
@@ -1,3 +1,6 @@
+:source: {{ cli }}.py
+:orphan:
+
```
Comment (samccann) [inline docs/templates/cli_rst.j2:2]: I'm unclear about this part ^^  why add :orphan: to the generated cli pages?

**Comment by samccann** [comment]
Thanks @Akasurde for getting this all working!  Just that one comment about the :orphan: addition and it's good to merge.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @Akasurde for this fix!

**Pull request merged**